pynamodb/attributes.py

```diff
@@ -77,8 +77,18 @@ def _initialize_attributes(cls):
         """
         cls._attributes = {}
         cls._dynamo_to_python_attrs = {}
-        for item_name, instance in six.iteritems(cls.__dict__):
-            if issubclass(type(instance), (Attribute,)):
+
+        for item_name in dir(cls):
```

Comment:
Discussed offline that we'd stick with using `dir` for now (what we've historically always used) and reconsider alternate approaches later